ui/src/utils/LLApiClient.js

```diff
@@ -96,6 +96,15 @@ class _LLApiClient {
     }
   });
 
+  requestAppAccess = ({ privacyPolicy, appName }) => post({
+    url: formatUrl(routes.REQUEST_APP_ACCESS),
+    headers: {
+      Authorization: `Bearer ${this.getToken()}`,
+      ContentType: 'application/json'
+    },
+    body: { privacyPolicy, appName },
```

Comment:
It's possible that Enterprise may require more info than this for other apps.

Reply:
I could change to sending an object containing that info? but not sure how to format the email template if doing that.

Reply:
Note to self: Discussed in person and decided to send config as JSON in email.